### PR TITLE
spring added to project

### PR DESCRIPTION
**Pull request by Rommelua**

### CODE REVIEW

File: src/main/java/com/dev/cinema/dao/impl/AbstractDao.java [hunk 1]
```diff
@@ -1,23 +1,31 @@
...
 import org.hibernate.Session;
+import org.hibernate.SessionFactory;
 import org.hibernate.Transaction;
 import org.hibernate.query.Query;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
 
 public class AbstractDao<T> {
     private static final Logger logger = LoggerFactory.getLogger(AbstractDao.class);
```
Comment (andromash) [inline src/main/java/com/dev/cinema/dao/impl/AbstractDao.java]: empty line :)

File: src/main/resources/db.properties [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# PostgreSQL properties
+db.driver=org.postgresql.Driver
+db.url=jdbc:postgresql://localhost:5432/cinema
+db.username=root
+db.password=1111
+# Hibernate properties
+hibernate.dialect=org.hibernate.dialect.PostgreSQL10Dialect
+hibernate.show_sql=false
```
Comment (andromash) [inline src/main/resources/db.properties:8]: isn't it false by default?

**Review by andromash — APPROVED** [review]
Nice!

File: src/main/resources/db.properties (hunk 2 above)
Comment (Rommelua) [reply src/main/resources/db.properties:8]: Yes. The reason - it will be easy to tune if you want to switch value.

**Review by Sofasmile — APPROVED** [review]

**Pull request merged**